src/Symfony/Bundle/FrameworkBundle/Command/EventDispatcherDebugCommand.php

```diff
@@ -120,6 +123,40 @@ protected function execute(InputInterface $input, OutputInterface $output): int
         return 0;
     }
 
+    public function complete(CompletionInput $input, CompletionSuggestions $suggestions): void
+    {
+        if ($input->mustSuggestArgumentValuesFor('event')) {
+            $dispatcherServiceName = $input->getOption('dispatcher');
+            if (!$this->dispatchers->has($dispatcherServiceName)) {
+                return;
+            }
+
+            $dispatcher = $this->dispatchers->get($dispatcherServiceName);
+            $suggestions->suggestValues(array_merge(
+                $dispatcher->getListeners()
+            ));
+
+            return;
+        }
+
+        if ($input->mustSuggestOptionValuesFor('dispatcher')) {
+            $suggestions->suggestValues(array_merge(
+                $this->dispatchers->getServiceIds(),
+                $this->dispatchers->getAliases()
```

Comment:
Not quite sure the dispatchers will be an instance of Container or ContainerBuilder?
Will there be aliases need to be supported on the completion?

Reply:
The Psalm CI job might fail on `getServiceIds` and `getAliases`. Maybe you can attach XDebug to your CLI job?